.circleci/config.yml

```diff
@@ -47,10 +47,9 @@ commands:
             git checkout -b "$branch_name"
             git commit -a -m "<<parameters.commit_message_prefix>> $(<<parameters.commit_message_command>>)"
             git push origin "$branch_name"
-            hub pull-request \
-              --no-edit \
-              --base="$CIRCLE_BRANCH" \
-              --labels="<<parameters.labels>>"
+            gh pr create\
+              --base "$CIRCLE_BRANCH" \
+              --label "<<parameters.labels>>"
```

Comment:
In here, we should pass the title, body, and type of the PR (submit, submit as draft. ...) otherwise, it would interactively ask in the cli. what are good values for these params?

Reply:
(I suppose this is no longer a concern)